README.md

```diff
@@ -4,25 +4,27 @@
 
 Implement the `App` with 2 pages and ability to select a person in the table.
 
-1. Create the `HomePage` available at `/` with just a title `Home Page`
-1. Create the `PeoplePage` available at `/people` with a title `People Page`
-1. Use [Navigate](https://reactrouter.com/docs/en/v6/components/navigate) component to redirect from `/home` to `/`;
+1. ✔ Create the `HomePage` available at `/` with just a title `Home Page`
+1. ✔ Create the `PeoplePage` available at `/people` with a title `People Page`
+1. ✔ Use [Navigate](https://reactrouter.com/docs/en/v6/components/navigate) component to redirect from `/home` to `/`; <!--- "404 There is no doc for components/navigate", link is invalid -->
```

Comment:
https://reactrouter.com/en/v6.3.0/api#navigate

Reply:
I suggested this should be updated in mate repository.